dev-env/bgp/frr/bgpd.conf

```diff
@@ -0,0 +1,39 @@
+! -*- bgp -*-
+!
+! BGPd sample configuration file
+!
+! $Id: bgpd.conf.sample,v 1.1 2002/12/13 20:15:29 paul Exp $
+!
+hostname bgpd
+password zebra
+!enable password please-set-at-here
+!
+router bgp 64512
```

Comment:
if you use the same bgp AS number for both peers it means you run iBGP (internal BGP), in scenarios I think is recommended to run eBGP (external BGP) because you have two different network realms or Autonomous Systems. Also, a "route learnt from an IBGP peer will not be advertised back to another iBGP neighbor by default",

Reply:
In this setup, I don't think propagating routes is necessary.  I can change one of the ASNs though if that makes more sense, though.

Reply:
is fine, just important to take into account if you want to grow it adding more peers